### ISSUE TITLE
Mismatch in reported testing accuracy and actual testing accuracy of the model on the sutd-traffic dataset

### ISSUE THREAD

**Issue by bennycortese** [XLiu443/Tem-adapter#4]
In the paper associated with the model and the pre-trained checkpoint, reported testing accuracy is 46% but when you run the training loop on the model from this repository, testing accuracy by default on the sutd-traffic dataset is 45.1%. I am not sure why this discrepancy occurs, thank you!

**Comment by CHENGY12** [comment]
Thanks for your questions!  It seems because of the environmental randomness.  I just re-trained using another independent device and obtained 45.7% accuracy.  We have released the checkpoint that was used in the paper [here](https://drive.google.com/drive/folders/1SplEKEjrp-Uw-PxziyBHvUuU-yQ0YevX).  In light of your suggestion, we will re-run the code 5 times and report an average performance and coresspooning std as clarification. Hope it can help.
